src/components/MovieItem/MovieItem.jsx

```diff
@@ -20,6 +23,7 @@ const MovieItem = ({ title, imageUrl }) => (
 );
 
 MovieItem.propTypes = {
+  movieObj: PropTypes.object.isRequired, // eslint-disable-line
```

Comment:
You should specify the shape of object. Take a look at `shape` https://facebook.github.io/react/docs/typechecking-with-proptypes.html